base_phone_popup/__openerp__.py

```diff
@@ -26,38 +26,10 @@
     'category': 'Phone',
     'license': 'AGPL-3',
     'summary': 'Pop-up the related form view to the user on incoming calls',
-    'description': """
-Base Phone Pop-up
-=================
-
-When the user receives a phone call, OpenERP can automatically open the
-corresponding partner/lead/employee/... in a pop-up without any action from the
-user.
-
-The module *web_action_request* can be downloaded with Mercurial:
-
-hg clone http://bitbucket.org/anybox/web_action_request
-
-It depends on 2 other modules, *web_longpolling* and *web_socketio*, that can
-be downloaded with this command:
-
-hg clone http://bitbucket.org/anybox/web_socketio
-
-You will find some hints in this documentation :
-https://bitbucket.org/anybox/web_action_request
-
-Warning : proxying WebSockets is only supported since Nginx 1.3.13 ; the
-feature provided by this module won't work with older versions of Nginx.
-
-TODO : document this new feature on the Akretion Web site :
-http://www.akretion.com/products-and-services/openerp-asterisk-voip-connector
-""",
     'author': "Akretion,Odoo Community Association (OCA)",
     'website': 'http://www.akretion.com/',
     'depends': ['base_phone', 'web_action_request'],
-    'data': [
-        'res_users_view.xml',
-        ],
+    'data': ['res_users_view.xml',],
```

Comment:
./base_phone_popup/__openerp__.py:32:34: E231 missing whitespace after ','

Please remove the `,` before `]`